### DISCUSSION TITLE
Laravel Pulse mongodb driver Support.

### DISCUSSION THREAD

**Discussion by mwakaambrose** [laravel/framework#49626] [General]
Does anyone know if there will be support for the mongodb driver? I recently ran into this.

<img width="1338" alt="Screenshot 2024-01-09 at 21 36 34" src="https://github.com/laravel/framework/assets/8328295/d12c5bd5-0f40-4b28-8a06-6f2d8e562aa8">

**Comment by shaedrich** [comment]
Well, there's [`jenssegers/laravel-mongodb-core`](https://github.com/jenssegers/laravel-mongodb-core) and [`mongodb/laravel-mongodb`](https://github.com/mongodb/laravel-mongodb). Does that help?

**Reply by reaves-tyler** [reply]
No, it doesn't 

> Pulse's first-party storage implementation currently requires a MySQL or PostgreSQL database. If you are using a different database engine, you will need a separate MySQL or PostgreSQL database for your Pulse data.

https://laravel.com/docs/10.x/pulse#installation

**Comment by yourjhay** [comment]
Anyone comes up with a solution to this ?
